docker/android-ndk.sh

```diff
@@ -6,26 +6,59 @@ set -euo pipefail
 # shellcheck disable=SC1091
 . lib.sh
 
-NDK_URL=https://dl.google.com/android/repository/android-ndk-r21d-linux-x86_64.zip
-
 main() {
-    local arch="${1}" \
-        api="${2}"
+    local arch="${1}"
+
+    if [[ "${ANDROID_SDK}" -eq 25 ]]; then
+        echo "Android SDK level 25 is not supported." 1>&2
+        exit 1
+    fi
 
-    install_packages curl unzip python
+    # python3 is still needed for newer NDK versions, just since it
+    # simplifies making symlinks even though the toolchain is prebuilt
+    install_packages curl python python3
+    get_ndk_info
+    if [[ "${NDK_VERSION}" -le 9 ]]; then
+        install_packages bzip2
```

Comment:
Very old versions package the binaries as a `tar.bz2` format, while anything Android NDK 10+ ships as a zipfile. As the NDK version 9 cannot be used with Android 5, in theory someone might not want to have an Android system and play around with very old versions, so I've left the logic here.